amundsen_application/static/js/components/common/ResourceListItem/UserListItem/index.tsx

```diff
@@ -22,34 +22,54 @@ class UserListItem extends React.Component<UserListItemProps, {}> {
     return `/user/${user.user_id}?index=${logging.index}&source=${logging.source}`;
   };
 
+  renderUserInfo = (user: UserResource) => {
+    const { role_name, team_name, user_id } = user;
```

Comment:
I rendered this with list items styled in a particular manner, but it could just as well be plain text with special characters as separators I think. Here's the mock:
![image](https://user-images.githubusercontent.com/1790900/67041208-fd50a100-f0d9-11e9-9c60-fd16ce8af425.png)

Thoughts?

Reply:
We could simplify with a `[user info].join('&bull;')` but we may need special formatting/styles in the future. Don't mind going either way on this.

Reply:
Alright, let's leave it and revisit if our design changes.